src/core/blockchain/ethereum/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { TransactionStatus } from '../../wallet/types';
+
+export const getEthStatusByCode = (status): TransactionStatus => {
```

Comment:
this should be renamed to `getTransactionStatusByCode()` and moved in `transactions.ts`.